### PR TITLE
Add sync dashboard

### PR DESCRIPTION
**Pull request by edmofro**

### CODE REVIEW

File: src/sync/SyncModal.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    alignSelf: 'center',
+  },
+  progressDescription: {
+    fontSize: 18,
+    color: 'white',
+    textAlign: 'center',
+    marginBottom: 10,
+  },
+  row: {
+    width: Dimensions.get('window').width / 3,
```
Comment (edmofro) [inline src/sync/SyncModal.js]: Hmmm...yeah maybe. I'll try it.

Comment (edmofro) [inline src/sync/SyncModal.js]: Nope it does need to be on the row. Ready for merge!

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "email": "info@sussol.net"
   },
   "name": "mSupplyMobile",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (nmadruga) [inline package.json]: Is this the right version? Aren't we going to release as 2.0.0?

Comment (edmofro) [inline package.json]: Yes you're right...will revert this

**Comment by edmofro** [comment]
Those are good points @nmadruga! I think they do improve the UX, but for now they are not critical to Timor rollout, so I will leave them for you or the guys to chase up when Timor is finished. I am on loan, and probably overdue to return to Tupaia, so will focus on the critical.

File: package.json (hunk 2 above)
Comment (Chris-Petty) [inline package.json]: We will release as 2.0.0, if the version number gets whack all we have to do is commit back to 2.0.0 and publish.

That said, I'd prefer something like 2.0.0-rc1 until release we increment the rc number.

Comment (edmofro) [inline package.json]: As I said when you made this comment earlier @Chris-Petty, I don't think you can do this with our smart android versioning atm. Can try to make it happen after looking at a couple of higher priorities, but have made v2.0.0 for now

**Comment by nmadruga** [comment]
@edmofro Yes, indeed. I'll do that now

**Pull request merged**